src/styles.scss

```diff
@@ -301,3 +301,13 @@ body > .app-container {
     width: 100%;
   }
 }
+
+.row {
+  margin-right: 0;
+  margin-left: 0;
+}
+
+.container-fluid {
+  padding-left: 0;
+  padding-right: 0;
+}
```

Comment:
Since these override `bootstrap` I think we should move these to their own file `bootstrap-overrides.scss` or something like it, that way its really easy to know where we should look if something brakes while upgrading bootstrap or the likes :)